src/code42cli/cmds/users.py

```diff
@@ -60,6 +61,50 @@ def list_users(state, org_uid, role_name, active, inactive, format):
         formatter.echo_formatted_dataframe(df)
 
 
+@users.command()
+@click.argument("username")
+@click.argument("role-name")
+@sdk_options()
+def add_role(state, username, role_name):
+    """Add the specified role to the specified username"""
```

Comment:
Maybe just `... specified user` at the end of the help text (rather than `username`)?  

Or `the user with the specified username`? 

Just because we are not adding a role to a username, but to the user themselves